gabarit/template_nlp/nlp_project/tests/test_model_huggingface.py

```diff
@@ -378,6 +394,66 @@ def test02_model_huggingface_fit(self):
             model.fit(x_train[:50], y_train_multi_fake[:50], x_valid=None, y_valid=None, with_shuffle=True)
         remove_dir(model_dir)
 
+
```

Comment:
Would diminish the batch size to be sure that the batches are not shuffled (or shuffled in the same way each time)